obs-packaging/scripts/pkglib.sh

```diff
@@ -13,7 +13,7 @@ OBS_SUBPROJECT="${OBS_SUBPROJECT:-alpha}"
 
 # BUILD OPTIONS
 BUILD_DISTROS=${BUILD_DISTROS:-Fedora_27 xUbuntu_16.04 CentOS_7}
-BUILD_ARCH=${BUILD_ARCH:-}x86_64
+BUILD_ARCH="$(uname -m)"
```

Comment:
Why not just `source ../scripts/lib.sh` to access [`$kata_arch_sh`](https://github.com/kata-containers/packaging/blob/master/scripts/lib.sh#L2) (like https://github.com/kata-containers/packaging/blob/master/obs-packaging/gen_versions_txt.sh#L16 does)?

Reply:
@jodh-intel : Because we are running this inside a "obs-kata" docker image with limited volume bindings. https://github.com/kata-containers/packaging/blob/f31f50a7eaee9702cf639474f5ccc6e16f7cfcad/obs-packaging/build_from_docker.sh#L46